### PR TITLE
Fixes xeno "pockets"

### LINKED ISSUE #10753: Xenomorph "pockets" are broken
If a xenomorph tries putting a facehugger in their pocket, the hugger simply falls to the ground. Later, due to reasons I can't pinpoint the game just begins to refuse letting you try placing the facehuggers there, like if the pockets were full.
Been happening this past week or two every time I become a xeno. Unfortunately I can't tell which day it stopped working since xenos aren't too common.

### PR DESCRIPTION
**Pull request by 9600bauds**
Plus some layer shit that Celt forgot to update
Fixes #10753

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
1000‰

**Comment by 9600bauds** [comment]
I thought this one was intended honestly
3 relatively minor issues in a week for a major refactor is pretty decent don't you think

File: code/modules/mob/living/carbon/martian/martian.dm [hunk 1]
```diff
@@ -115,6 +115,7 @@
 		update_inv_head(redraw_mob)
 
 	W.layer = 20
+	W.plane = PLANE_HUD
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/martian/martian.dm:118]: Why is this stuff not a proc then people would stop forgetting to do it all aaaaa

**Comment by clusterfack** [comment]
PLANES

**Pull request merged**